src/tools/web/README.md

```diff
@@ -0,0 +1,37 @@
+# elektra web
+
+_an API and web user interface to remotely manage multiple elektra instances_
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon ([`elektrad`](elektrad/))
+ * a single cluster management server to communicate with the elektra daemons ([`clusterd`](clusterd/))
+ * a client (web browser) that accesses the Web UI on the cluster management server ([`client`](client/))
```

Comment:
the client actually doesn't access elektrad directly, clusterd acts as a kind of router that maintains instances (this makes it easy to have a client that can be accessed from various devices, otherwise I'd have to store the connection info for clusterd/elektrad instances in localstorage, cookies or something similar). this way all you have to provide in the client is the host of the clusterd instance.